### PR TITLE
feat: create a group conversation (AR-908)

### PR DESCRIPTION
**Pull request by trOnk12**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-908" title="AR-908" target="_blank">StoryAR-908</a>  Start a new group conversation
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

create a group conversation

### Note

This should be merged after https://github.com/wireapp/wire-android-reloaded/pull/502
because this is a continued work based on that PR 

### Attachments (Optional)

group_conv

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-508/1/) **failed**.

**Comment by gongracr** [comment]
I think you should resolve the conflicts first so that we can see the most updated state of the PR

File: app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt [hunk 1]
```diff
@@ -47,7 +81,19 @@ class OtherUserProfileScreenViewModel @Inject constructor(
     }
 
     fun openConversation() {
-        //TODO: fire a use case
+        viewModelScope.launch {
+            when (val result = getOrCreateOneToOneConversation(userId)) {
+                is CreateConversationResult.Failure -> appLogger.d(("Couldn't retrieve or create the conversation"))
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt:86]: Maybe add a TODO to handle properly the creation error?

**Review by gongracr — APPROVED** [review]
There seems to be some outdated code that should be overriden by the current state of `develop`. Just make sure you solve the conflicts correctly.

Other than that awesome work as usual! 💪 getting closer to the point where there is no need to keep using the legacy android app 😻

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-508/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-508/3/) **failed**.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-508/4/) **failed**.

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-508/5/) **failed**.

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-508/6/) **failed**.

**Comment by AndroidBob** [comment]
Build [7](https://10.10.124.134/job/wire-android-reloaded/job/PR-508/7/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3140265)(3140265)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3140265)(3140265)-dev-release.apk)

**Pull request merged**